packages/kit/src/core/config/index.js

```diff
@@ -37,6 +37,34 @@ export function load_template(cwd, config) {
 	return fs.readFileSync(template, 'utf-8');
 }
 
+/**
+ * Loads the error page (src/error.html by default) if it exists.
+ * Falls back to a generic error page content.
+ * @param {string} cwd
+ * @param {import('types').ValidatedConfig} config
+ */
+export function load_error_page(cwd, config) {
+	const { errorPage } = config.kit.files;
+	const relative = path.relative(cwd, errorPage);
+
+	if (fs.existsSync(errorPage)) {
+		return fs.readFileSync(errorPage, 'utf-8');
+	} else {
+		console.log(`${relative} does not exist. Using generic error page instead.`);
```

Comment:
I made it so that in case `src/error.html` (or whatever) doesn't exist we only output a log and fall back to a generic error page instead. This way not everyone needs to have this if they don't want to (or need to, in case of SPAs). Alternatives:
- be completely silent / no logging
- throw